cypress/integration/transfer/drag-and-drop-items.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Drag and drop selected items
```

Comment:
I'm wondering if it makes sense to allow the reordering functionality if there's only one item.
We could change the first scenario to `two or more` and add an additional scenario that states that reordering is not available if there's only one item in the list. The currently second scenario would need to change to `three or more` then to make sense.

Reply:
Agreed, it's best to be specific and cover all the use cases. I've modified the existing scenarios and added a new one for disabling reordering for a single item.